.github/workflows/pkgci_sharktank_regression.yml

```
@@ -0,0 +1,97 @@
+# Copyright 2024 The IREE Authors
+#
+# Licensed under the Apache License v2.0 with LLVM Exceptions.
+# See https://llvm.org/LICENSE.txt for license information.
+# SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+
+name: PkgCI Sharktank Regression Tests
+on:
+  workflow_call:
+    inputs:
+      artifact_run_id:
+        type: string
+        default: ""
+  workflow_dispatch:
+    inputs:
+      artifact_run_id:
+        type: string
+        default: ""
+
+jobs:
+  sharktank_regresion_tests:
+    name: "sharktank_regresion_tests :: ${{ matrix.name }}"
```

typo in "regression"
```suggestion
  sharktank_regression_tests:
    name: "sharktank_regression_tests :: ${{ matrix.name }}"
```